web/src/lib/components/pages.tsx

```diff
@@ -324,124 +326,198 @@ export default class Pages extends Component<PagesProps, PagesState> {
-                <p class="strong">Help</p>
-              </a>
-              <a id="contribute"
-                 target="_blank" href="https://github.com/mozilla/voice-web">
-                <Icon type="github" />
-                <p class="strong">Contribute</p>
-              </a>
-              <a id="discourse"
-                 target="blank" href="https://discourse.mozilla-community.org/c/voice">
-                <Icon type="discourse" />
-                <p class="strong">Community</p>
-              </a>
+                api={this.props.api}
+                user={this.props.user}
+              />
+              <Record
+                active={this.isPageActive(URLS.RECORD)}
+                api={this.props.api}
+                onRecord={this.onRecord}
+                onRecordStop={this.onRecordStop}
+                onRecordingSet={this.sayThanks}
+                onVolume={this.onVolume}
+                onSubmit={this.uploadRecordings}
+                onDelete={this.clearRobot}
+                navigate={this.props.navigate}
+                user={this.props.user}
+              />
+              <Listen
+                active={this.isPageActive(URLS.LISTEN)}
+                navigate={this.props.navigate}
+                api={this.props.api}
+                user={this.props.user}
+              />
+              <Profile
+                user={this.props.user}
+                active={this.isPageActive(URLS.PROFILE)}
+              />
+              <FAQ active={this.isPageActive(URLS.FAQ)} />
+              <Privacy active={this.isPageActive(URLS.PRIVACY)} />
+              <Terms active={this.isPageActive(URLS.TERMS)} />
+              <NotFound active={this.isNotFoundActive()} />
             </div>
-          </div>
-          <div id="moz-links">
-            <div class="content">
-              <Logo navigate={this.props.navigate}/>
-              <div class="links">
-                <p>
-                  <a onClick={this.linkNavigate} href="/privacy">Privacy</a>
-                  <a onClick={this.linkNavigate} href="/terms">Terms</a>
-                  <a target="_blank" href="https://www.mozilla.org/en-US/privacy/websites/#cookies">Cookies</a>
-                  <a onClick={this.linkNavigate} href="/faq">FAQ</a>
-                </p>
-                <p>Content available under a&nbsp;<a target="_blank" href="https://www.mozilla.org/en-US/foundation/licensing/website-content/">Creative Commons license</a></p>
+            <footer>
+              <div id="help-links">
+                <div class="content">
+                  <a id="help" onClick={this.linkNavigate} href="/faq">
+                    <Icon type="help" />
+                    <p class="strong">Help</p>
+                  </a>
+                  <a
+                    id="contribute"
+                    target="_blank"
+                    href="https://github.com/mozilla/voice-web">
+                    <Icon type="github" />
```

Comment:
This would be easier to read if we used the Prettier default behavior for `jsxBracketSameLine`, but in general I think the closing bracket on the same line is more readable. Indeed, React core sets jsxBracketSameLine to true.